build_extra/toolchain/win/BUILD.gn

```diff
@@ -111,7 +111,7 @@ template("msvc_toolchain") {
 
       # lld-link includes a replacement for lib.exe that can produce thin
       # archives and understands bitcode (for lto builds).
-      lib = "$prefix/$lld_link /lib /llvmlibthin"
+      lib = "$prefix/$lld_link /lib"
```

Comment:
Add a gn flag for this, and we should send the patch upstream. I'd very much like to not diverge from chrome's build.

Reply:
> Add a gn flag for this, and we should send the patch upstream. 
> I'd very much like to not diverge from chrome's build.

Those comments are in direct conflict with one another. This BUILD.gn is derived from chromiums and modified with some simple sed edits. You can opt to either full-on fork it or you can live without the flag for now.